lib/errors/index.ts

```diff
@@ -139,12 +149,46 @@ export class ArsenalError extends Error {
             const error = value[1];
             const { code, description } = error;
             const get = () => new ArsenalError(name, code, description);
-            Object.defineProperty(errors, name, { get });
+            // Perf improvement for the ok error.
+            // As it's used mainly for HTTP response using the code and message only.
+            // Return the same instance as no stack trace or other property are needed
+            if (name === 'ok') {
+                Object.defineProperty(errors, name, { value: get() });
+            } else {
+                // Ideally for other errors caching the instance outside request handling
+                // would be beneficial if .customizeDescription is used or if
+                // no stack trace or additional field is required
+                Object.defineProperty(errors, name, { get });
+            }
         });
         return errors as Errors
     }
+
+        /**
+         * Like errors but generate singleton instances of errors
+         * To use before .customizeDescription to avoid intermediate instance.
+         * Adds +0.2 MB in heap at start
+         */
+        static errorInstances() {
```

Comment:
Should we enforce it? Or warn the contributor?

Reply:
Ideally I'd prefer we enforce it, and when I bump arsenal I'd replace any occurence of of `errors.xxx.customizeDescription` or `errors.xxx.code` with this so we have the good pattern.

Reply:
How to enforce it?

Reply:
Right now there is no automated way of doing it. Just during review.